sdk/maps/azure-maps-render/README.md

```diff
@@ -0,0 +1,158 @@
+# Azure Resource Manager Render client library for Java
+
+Azure Resource Manager Render client library for Java.
+
+This package contains Microsoft Azure SDK for Render Management SDK. Azure Maps Render REST APIs. Package tag 2.1-preview. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
```

Comment:
1. Package tag is wrong.
2. Can we use the similar description (first section) like [storage](https://github.com/khmic5/azure-sdk-for-java-1/tree/maps/onboarding/search-route-render/sdk/storage/azure-storage-blob)?

Azure Maps Render is Microsoft's ......
[Source code](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/src) | [API reference documentation](https://azure.github.io/azure-sdk-for-java/) | [REST API documentation](https://docs.microsoft.com/rest/api/storageservices/blob-service-rest-api) | [Product documentation](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-overview) | [Samples](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/src/samples)